README.md

```diff
@@ -46,6 +46,10 @@ If you want to use a package manager:
 - [Snap](https://snapcraft.io/) users can use `sudo snap install helm
   --classic`.
 
+If you want to install helm from script:
```

Comment:
I would just combine the item in the bullet point up to here and say

```
If you want to install the Helm using a script, follow [the instructions](link)...
```

Also, make sure to capitalize Helm here unless you mean the binary, in which case write it as `helm`. Same goes for the use of `Helm` below. It likely should be `helm`